scripts/us_cdc/environmental_health_toxicology/README.md

```diff
@@ -47,80 +47,101 @@ These data were collected as part of the [CDC National Environment Public Health
 
-[`PM25CountyPollution.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/PM25CountyPollution.tmcf)
+#### tMCFs
+[`PalmerDroughtSeverityIndex.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/precipitation/tMCFs/PalmerDroughtSeverityIndex.tmcf)
 
-[`StandardizedPrecipitationEvapotranspirationIndex.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/StandardizedPrecipitationEvapotranspirationIndex.tmcf)
+[`StandardizedPrecipitationEvapotranspirationIndex.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/precipitation/tMCFs/StandardizedPrecipitationEvapotranspirationIndex.tmcf)
 
-[`StandardizedPrecipitationIndex.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/StandardizedPrecipitationIndex.tmcf)
+[`StandardizedPrecipitationIndex.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/precipitation/tMCFs/StandardizedPrecipitationIndex.tmcf)
 
 ### Import Procedure
 
 #### Testing
 
-##### Test Air Quality Data Cleaning Script
+`@input_file` - path to the test input csv file to be cleaned
+
+`@expected_output_file` - path to cleaned csv file with the expected output
 
 To test the air quality data cleaning script, run:
 
 ```bash
-$ python3 parse_air_quality_test.py
+$ python3 parse_air_quality_test.py input_file expected_output_file
 ```
 
-The expected output of this test can be found in [`small_Ozone_County_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/test_data/small_Ozone_County_expected.csv).
-
-##### Test Precipitation Index Data Cleaning Script
+The test files for the 4 types of air quality datasets and their paired expected output files can be found [here](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/test_files/air_quality).
 
 To test the precipitation index data cleaning script, run:
 
 ```bash
-$ python3 parse_precipitation_index_test.py
+$ python3 parse_precipitation_index_test.py input_file expected_output_file
 ```
 
-The expected output of this test can be found in [`small_Palmer_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/test_data/small_Palmer_expected.csv).
+The test files for the 3 types of precipitation datasets and their paired expected output files can be found [here](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/test_files/precipitation).
```

Comment:
This entire section can probably be skipped, if we just list the pairs of input/expected CSV files, and the standard unittest mechanism will auto-run it.

Reply:
removed